raiden/tests/integration/network/transport/rif_comms/test_client.py

```diff
@@ -90,223 +101,207 @@ def test_connect_peers():
             node.stop()
 
 
-@pytest.mark.parametrize("amount_of_nodes", [1])
-def test_subscribe_to_invalid(comms_nodes):
-    client = comms_nodes[1].client
-    unregistered_address = generate_address()
-
-    # no subscriptions should be present
-    assert client._is_subscribed_to(unregistered_address) is False
-
-    # subscribe to unregistered address
-    topic_id, _ = client.subscribe_to(unregistered_address)
-
-    # check subscription again
-    # FIXME: this should probably throw an exception rather than an empty result
-    assert topic_id is ""
-    assert client._is_subscribed_to(unregistered_address) is False
+def test_send_message_shutdown():
```

Comment:
please keep the original order for the tests (e.g.: message-sending tests should be grouped together instead of spread out over the module)

Reply:
done